mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -223,6 +267,29 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
             .disposeOnClear()
     }
 
+    /**
+     * Similar to [execute]. This subscribes to an Observable, provides a reducing function to update state when the observable passes a value,
+     * and registers the subscription to be disposed when the ViewModel is cleared.
+     *
+     * This function should be preferred over calling [Observable.subscribe] directly so that disposal is handled for you, and so setting state
+     * is simplified.
+     */
+    fun <T> Observable<T>.executeWithoutAsync(stateReducer: S.(value: T) -> S): Disposable {
```

Comment:
What about just calling this `subscribe`? Or something with that terminology. MvRx adds execute terminology and it has been working great. Not sure if that should be used here.

Reply:
The reason I prefixed this with execute was to include it in autocomplete. If people are used to using execute with mvrx, then `executeWithoutAsync` is discoverable